### PR TITLE
Gnw changes

### PR DESCRIPTION
**Pull request by SkewedAskew**
Rounding out that which has fallen flat

Changelog: [Gnw_changes.txt](https://github.com/HDR-Development/HewDraw-Remix/files/9795398/Gnw_changes.txt)

Hitbox Comparisons: https://imgur.com/a/2LdYKBY

### CODE REVIEW

**Comment by yoonikoo** [comment]
DAIR still does not ledge cancel. BAIR does indeed combo into BAIR at low percents which is great for racking it up. The single hit doot has pretty unique properties now. Low percents, on a weak it doot, it will juggle just enough for GnW to get a grab and go for mixups from there. Also, if performed fast enough, fire from chef pan will lead into a regrab at low percents. GnW definitely has a lot more mixups going for him, as the doot now serves as a juggle tool and FAIR having a single frame of a reverse HB. However, the doot hitbox does not fit the visuals and should be adjusted accordingly.

**Comment by yoonikoo** [comment]
2022100621183000-0E7DF678130F4F0FA2C88AE72B47AFDF

**Comment by yoonikoo** [comment]
https://user-images.githubusercontent.com/105788245/194467521-443b5d21-e5e2-4fa3-b899-1f2bb6592d31.mp4

**Comment by SkewedAskew** [comment]
Dair is a pesky issue but will get fixed one way or another. Uair's hitbox has since been corrected

Edit: fixed both

File: fighters/gamewatch/src/status.rs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+//         // set the callbacks on fighter init
+//         if fighter.kind() == *FIGHTER_KIND_GANON {
+//             fighter.global_table[globals::USE_SPECIAL_N_CALLBACK].assign(&L2CValue::Ptr(should_use_special_n_callback as *const () as _));
+//             fighter.global_table[globals::STATUS_CHANGE_CALLBACK].assign(&L2CValue::Ptr(change_status_callback as *const () as _));   
+//         }
+//     }
+// }
+
+#[status_script(agent = "gamewatch", status = FIGHTER_STATUS_KIND_LANDING_ATTACK_AIR, condition = LUA_SCRIPT_STATUS_FUNC_STATUS_PRE)]
+pub unsafe fn landing_attack_air_pre(fighter: L2CFighterCommon) -> L2CValue {
```
Comment (WuBoytH) [inline fighters/gamewatch/src/status.rs]: This should be moved into `landing_attack_air.rs` and replace the  one that's already there

**Review by WuBoytH — COMMENTED** [review]
hopefully my comment showed up

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Review by WuBoytH — APPROVED** [review]

**Pull request merged**